content/examples/_index.md

```diff
@@ -15,33 +15,33 @@ The default notification type is **short**
 
 #### Resource created
 
-![](/images/create_sh.png "Slack notiftype=short")
-![](/images/create.png "Slack notiftype=long")
+![](/images/create_sh.png "Slack notifyType=short")
```

Comment:
While it's not really important, please also change `notifyType` to `notification.type` for all the images. Thanks!